tests/battery_module.sh

```diff
@@ -10,3 +10,6 @@ tmux source "${script_dir}/../catppuccin_tmux.conf"
 
 print_option @catppuccin_battery_icon | grep -q "#{battery_icon}" ||
   echo "@catppuccin_status_battery expanded #{battery_icon} in @catppuccin_battery_icon"
+
+print_option @catppuccin_battery_icon | grep -q "#{l:#{battery_icon}}" ||
```

Comment:
Can you add an expected output and match against that instead? I don't trust grep as much, it's not clear what is actually being printed by the option

Reply:
Added an expected output match, had to resort to defining a newline char for expansion in the match because `print_option` adds a newline.

Reply:
I'm not seeing a change to `tests/battery_module_expected.txt`

Reply:
Also for reference I'm pretty sure `\n` works

Reply:
Ah, I misunderstood the ask. I'll go take a look at the expected file.

Reply:
Is this how you use the `_expected.txt` file?